pubspec.yaml

```diff
@@ -1,13 +1,11 @@
 name: ffi
-version: 0.3.0-nullsafety.2
+version: 1.0.0-nullsafety.0
 homepage: https://github.com/dart-lang/ffi
 description: Utilities for working with Foreign Function Interface (FFI) code.
 
 environment:
   sdk: '>=2.12.0-259.9.beta <3.0.0'
 
-# dependencies:
-
 dev_dependencies:
   pedantic: ^1.10.0-nullsafety
```

Comment:
Can we roll these deps?

https://pub.dev/packages/pedantic/versions/1.10.0
https://pub.dev/packages/test/versions/1.16.0

Reply:
done